Stack/Opc.Ua.Core/Security/Certificates/CertificateFactory.cs

```diff
@@ -226,7 +226,8 @@ public static X509Certificate2 Load(X509Certificate2 certificate, bool ensurePri
         ushort lifetimeInMonths,
         ushort hashSizeInBits,
         bool isCA,
-        X509Certificate2 issuerCAKeyCert)
+        X509Certificate2 issuerCAKeyCert,
+        byte[] publicKey = null)
```

Comment:
Agreed. This will be refactored when we build the Crypto Service Provider interface and refactor the cert factory class.